README.md

```diff
@@ -9,28 +9,18 @@ Clarifai Python SDK</a>
 
 
 <p align="center">
-  <a href="https://clarifaicommunity.slack.com/" target="_blank"> <img src="https://img.shields.io/badge/slack-chat-green.svg?logo=slack&color=4ec528" alt="Slack">
+  <a href="https://discord.gg/M32V7a7a" target="_blank"> <img src="https://img.shields.io/badge/discord-chat-green.svg?logo=discord&color=4ec528" alt="Discord">
+  </a>
+  <a href="https://pypi.org/project/clarifai" target="_blank"> <img src="https://img.shields.io/pypi/dm/clarifai" alt="PyPI - Downloads">
   </a>
-
 </p>
 
 
 
 
 This is the official Python client for interacting with our powerful [API](https://docs.clarifai.com). The Clarifai Python SDK offers a comprehensive set of tools to integrate Clarifai's AI platform to leverage computer vision capabiities like classification , detection ,segementation and natural language capabilities like classification , summarisation , generation , Q&A ,etc into your applications. With just a few lines of code, you can leverage cutting-edge artificial intelligence to unlock valuable insights from visual and textual content.
 
----
-**Website**: [https://www.clarifai.com](https://www.clarifai.com/)
-
-**Demo**: [https://clarifai.com/demo](https://clarifai.com/demo)
-
-**Sign up for a free Account**: [https://clarifai.com/signup](https://clarifai.com/signup)
-
-**Developer Guide**: [https://docs.clarifai.com](https://docs.clarifai.com/)
-
-**Clarifai Community**: [https://clarifai.com/explore](https://clarifai.com/explore)
-
-**Python SDK Docs**: [https://docs.clarifai.com/python-sdk/api-reference](https://docs.clarifai.com/python-sdk/api-reference)
+[Website](https://www.clarifai.com/) | [Demo](https://clarifai.com/demo) | [Signup for a Free Account](https://clarifai.com/signup) | [API Docs](https://docs.clarifai.com/) | [Clarifai Community](https://clarifai.com/explore) | [Python SDK Docs](https://docs.clarifai.com/python-sdk/api-reference) | [Examples](https://github.com/Clarifai/examples) | [Colab Notebooks](https://github.com/Clarifai/colab-notebooks)
 
```

Comment:
I wonder if we should point to `Colab Notebooks` as they doesn't use SDK yet

Reply:
It's fine. At least it will be on the radar. How i see it, it shouldn't be SDK or bust. Users currently are using python GRPC can still use our API. Moreover, we direct users to our API docs which also does not provide SDK examples, so it's consistent so far.